CalibCalorimetry/EcalTrivialCondModules/src/EcalTrivialConditionRetriever.cc

```diff
@@ -3271,3 +3292,77 @@ EcalTrivialConditionRetriever::produceEcalSamplesCorrelation( const EcalSamplesC
        back_inserter(ipar->EEG1SamplesCorrelation));
   return ipar;
 }
+
+
+
+
+
+
+
+
+std::unique_ptr<EcalSimPulseShape>
+EcalTrivialConditionRetriever::getEcalSimPulseShapeFromConfiguration 
+( const EcalSimPulseShapeRcd& )
+{
+  std::cout<< "Produce EcalSimPulseShape set time interval="<<sim_pulse_shape_TI<< " ns"<< std::endl; 
+
+  auto result = std::make_unique<EcalSimPulseShape>();
+ 
+
+  result->time_interval = sim_pulse_shape_TI;
+
+  std::vector<double> EBshape_;
```

Comment:
don't use a trailing `_` for local variables

Reply:
OK